config/os/openafs_client.pan

```diff
@@ -0,0 +1,4 @@
+unique template config/os/openafs_client;
+
+include { 'rpms/openafs-client' };
```

Comment:
As a general remark, you should adopt the new `include` syntax introduced by panc v10 which is simpler:
```
include 'rpms/openafs-client';
```